snap/snapcraft.yaml

```diff
@@ -75,8 +74,13 @@ parts:
       - "-DCMAKE_SKIP_RPATH=TRUE"
       - "-DCMAKE_PREFIX_PATH=$(echo $SNAPCRAFT_CMAKE_ARGS | awk -F= '{printf(\"%s/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/cmake/Qt5\", $2)}')"
       - "--event-handlers=console_direct+"
+      - "--executor sequential"
```

Comment:
Why `sequential`?

Reply:
Because I needed determinism when debugging the building issue with standard GitHub runners. And I kinda liked it since I can compare logs and timings. I don't think it hurts the speed that much given how the codebase is structured. After we merge I can try to remove it and compare the timings

Reply:
Gotcha.
Indeed it would be nice to see how much time can be saved having the build be parallel.

Reply:
removed